src/mame/includes/asteroid.h

```diff
@@ -9,6 +9,28 @@
 #include "sound/discrete.h"
 #include "video/avgdvg.h"
 
+#define ASTEROID_SAUCER_SND_EN      NODE_01
+#define ASTEROID_SAUCER_FIRE_EN     NODE_02
+#define ASTEROID_SAUCER_SEL         NODE_03
+#define ASTEROID_THRUST_EN          NODE_04
+#define ASTEROID_SHIP_FIRE_EN       NODE_05
+#define ASTEROID_LIFE_EN            NODE_06
+#define ASTEROID_NOISE_RESET        NODE_07
+
+#define ASTEROID_THUMP_EN           NODE_08
+#define ASTEROID_THUMP_DATA         NODE_09
+#define ASTEROID_EXPLODE_DATA       NODE_10
+#define ASTEROID_EXPLODE_PITCH      NODE_11
+
+#define ASTEROID_NOISE              NODE_20
+#define ASTEROID_THUMP_SND          NODE_21
+#define ASTEROID_SAUCER_SND         NODE_22
+#define ASTEROID_LIFE_SND           NODE_23
+#define ASTEROID_SAUCER_FIRE_SND    NODE_24
+#define ASTEROID_SHIP_FIRE_SND      NODE_25
+#define ASTEROID_EXPLODE_SND        NODE_26
+#define ASTEROID_THRUST_SND         NODE_27
```

Comment:
Is there really no way to put this stuff back inside the audio device and expose it with write line members or something?